src/components/GoodsData.ts

```diff
@@ -0,0 +1,51 @@
+import goodsData from '../modules/goods';
+
+export default class GoodsData {
+  static getAllPrice(): number[] {
+    const allPrices: number[] = [];
+    for (let i = 0; i < goodsData.products.length; i++) {
+      allPrices.push(goodsData.products[i].price);
+    }
+    return allPrices;
+  }
```

Comment:
try to use this 
 `static getAllPrice(): number[] {
    return goodsData.products.map(product => product.price);
  }`